controllers/apply/form-router-next/index.js

```diff
@@ -127,6 +130,30 @@ function initFormRouter({
         }
     }
 
+    /**
+     * Application seed endpoint
+     * Allows generation of seed applications in test environments
+     */
+    if (features.enableSeeders) {
+        router.post('/seed', async (req, res) => {
+            const application = await PendingApplication.createNewApplication({
+                formId: formId,
+                userId: req.body.userId,
+                customExpiry: req.body.expiresAt
+            });
+
+            if (expiryEmailPeriods) {
```

Comment:
Should this code be lifted up into `createNewApplication`? Do we always create queue items when creating an application?

Reply:
This code will be deleted shortly so for now let's leave this as a separate explicit thing (and see if the Standard form will need this functionality, in which case we'll know if there's a parallel between application/queue creation).